lib/waterline/query/dql/create.js

```diff
@@ -215,17 +215,10 @@ function createValues(valuesObject, cb) {
       return after(parentModel.toObject());
     });
 
-
     function after(values) {
```

Comment:
Looks like this might be only getting used on L215, where it was converted _from_ a model instance to an object right before this. If that's true, I think we can remove this fn altogether along with the `toObject` call above?